batch7crud-roll5/src/main/java/com/lftechnology/batch7crud/services/EmployeeServiceImpl.java

```diff
@@ -1,12 +1,18 @@
 package com.lftechnology.batch7crud.services;
 
 import java.util.List;
+
+import javax.validation.ValidationException;
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove this unused import 'javax.validation.ValidationException'. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://nemo.sonarqube.org/coding_rules#rule_key=squid%3AUselessImportCheck)